### PR TITLE
stolostron/backlog#18017 Add feature list

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

File: clusters/create_infra_env.adoc [hunk 1]
```diff
@@ -8,6 +8,17 @@ You can use the {product-title} console to create an infrastructure environment
...
+* Late binding: Enable the host to be booted by an infrastructure administrator, and the creator of a cluster can bind a cluster to that host at a later time. The cluster creator does not have to have administrator privileges to the infrastructure when you use late binding.
+* Dual stack: Deploy clusters that have both IPv4 and IPv6 addresses. Dual stack uses the `OVN-Kubernetes` networking implementation to support multiple subnets. 
+* Add remote worker nodes: Add remote worker nodes to your clusters after they are created and running, which provides flexibility of adding nodes in other locations for backup purposes.
+* Add day2 worker nodes: Add worker nodes to a running cluster.
+* Upgrade {product-title-short} version 2.3 to {product-title-short} 2.4 with Assisted Installer static IP addressing: Use the Assisted Installer to upgrade your remote clusters to {product-title-short} version 2.4.
```
Comment (chadcrum) [inline clusters/create_infra_env.adoc]: @cadawson (Apologies for the delay) -  I'm confused about this Upgrade line - Is there an epic or other source it comes from? Assisted Installer can do deployments and add workers, but nothing with upgrades.

Comment (cadawson) [reply clusters/create_infra_env.adoc]: Thanks @chadcrum . No worries. I pulled it from the comments here: https://github.com/stolostron/backlog/issues/18017#issuecomment-973216155. I might not be presenting it correctly, or understanding what it is. Let me know if it should be removed or changed. Thanks!

Comment (chadcrum) [reply clusters/create_infra_env.adoc]: I see it @cadawson - thanks! 

To clarify: Upgrading ACM on the hub cluster from 2.3 -> 2.4 also upgrades the Infrastructure Operator / Operand (Assisted Service) on the hub from 2.3 to the 2.4 bundled version. The spoke clusters are not upgraded.

Comment (cadawson) [reply clusters/create_infra_env.adoc]: Thanks!

**Review by berenss — COMMENTED** [review]
This is great and much needed for both 2.4 and 2.5.  MCE 2.0 / ACM 2.5 represents the GA release of this delivery.

Suggest we link out to the latest OCP 4.10 doc for ZTP.

We are missing a bullet about Static IP using nmstate- this is a big feature for customers.

I do not see any notes about what hardware we can interface with. BMC information would be nice to have as well.

File: clusters/create_infra_env.adoc [hunk 2]
```diff
@@ -8,6 +8,17 @@ You can use the {product-title} console to create an infrastructure environment
...
+* Late binding: Enable the host to be booted by an infrastructure administrator, and the creator of a cluster can bind a cluster to that host at a later time. The cluster creator does not have to have administrator privileges to the infrastructure when you use late binding.
+* Dual stack: Deploy clusters that have both IPv4 and IPv6 addresses. Dual stack uses the `OVN-Kubernetes` networking implementation to support multiple subnets. 
+* Add remote worker nodes: Add remote worker nodes to your clusters after they are created and running, which provides flexibility of adding nodes in other locations for backup purposes.
+* Add day2 worker nodes: Add worker nodes to a running cluster.
+* Upgrade {product-title-short} version 2.3 to {product-title-short} 2.4 with Assisted Installer static IP addressing: Upgrading {product-title-short} on the hub cluster from version 2.3 to 2.4 upgrades the infrastructure operator and infrastructure operand, or the assisted service, on the hub cluster from version 2.3 to the 2.4 bundled version. The managed clusters are not automatically upgraded.
```
Comment (chadcrum) [inline clusters/create_infra_env.adoc]: Upgrade {product-title-short} version 2.3 to {product-title-short} 2.4: Upgrading {product-title-short} on the hub cluster from version 2.3 to 2.4 upgrades the Infrastructure Operator on the hub cluster from version 2.3 to the 2.4. The managed clusters are not automatically upgraded.

@cadawson ^^ Actually I think this wording is better

Comment (berenss) [reply clusters/create_infra_env.adoc]: I don't understand what this comment is for. We are releasing version 2.5, so would this need to be a statement about hub upgrade of 2.4 to 2.5?

**Comment by cadawson** [comment]
Thanks @berenss and @chadcrum . I added a bullet for NMState that hopefully covers that part. Do you have any information about the hardware and BMC, if you want to include that? I have not seen anything related to those. Thanks!

**Comment by cadawson** [comment]
Hi @berenss and @chadcrum . I made a couple of updates for Scott's comments. Does this look ok? Thanks!

**Comment by cadawson** [comment]
@berenss and @chadcrum: Can you check out the latest version of this and let me know which hardware and BMC information you would like to include? Thanks!

**Review by berenss — APPROVED** [review]
looks good!

**Review by filanov — APPROVED** [review]

**Pull request merged**